### PR TITLE
Feature/card

### PR DESCRIPTION
**Pull request by oscarfgq**
## Changes in this PR
- Created new component `Card`.
- Created new component `CardList`.
- Added stories for both components to storybook.
- Added unit tests for both components.

### CODE REVIEW

File: src/components/__tests__/CardList.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+describe('Card Component', () => {
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<CardList />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+ 
+  it('should render children', () => {
+    const wrapper = mount(
+      <CardList>
```
Comment (diegolanda) [inline src/components/__tests__/CardList.js]: This should be styled now

File: stories/index.js [hunk 2]
```diff
@@ -48,6 +48,42 @@ storiesOf('Web UI Components', module)
     ))
   );
```
Comment (diegolanda) [inline stories/index.js]: can't do an `.add` after this, you have a `;` storybook would fail

File: stories/index.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { storiesOf, addDecorator } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { withInfo } from '@storybook/addon-info';
 
-import { ImageTag, TechIcon, TechIconList } from '../src/components';
+import { ImageTag, TechIcon, TechIconList, Card, CardList } from '../src/components';
```
Comment (diegolanda) [inline stories/index.js]: try to keep alphabetical order

**Review by diegolanda — CHANGES_REQUESTED** [review]

File: stories/index.js (hunk 3 above)
Comment (oscarfgq) [reply stories/index.js]: Right. Done.

**Review by diegolanda — APPROVED** [review]

**Pull request merged**